nodes/DictComp.py

```diff
@@ -1 +1,20 @@
+"""
+DictComp astroid node
+
+A DictComp node represents dictionary comprehensions in Python.
+
+Attributes:
+    - key         (node)
+        - The part that will be evaluated for each item.
+    - value       (node)
+        - The part that will be evaluated for each item.
+    - generators  (List[Node])
```

Comment:
If `generators` must be a list of Comprehension nodes, the proper type is `List[Comprehension]`